### DISCUSSION TITLE
Toggle/reset local offset

### DISCUSSION THREAD

**Discussion by Widosm** [ppy/osu#17884] [Ideas] [answered]
Everytime I want to reset the local offset to 0 (usually after changing global offset), it's annoying to have to drag the small slider back to 0. It would be nice to have a "Toggle local offset on/off" button, or maybe one that just resets it back to 0.

**Comment by Theighlin** [comment, marked as answer]
To reset it to 0 (the default) you can click the yellow circle that appears on the left of the slider. This also applies for other settings accross the game.

**Reply by AuracleTech** [reply]
Thank you for this
I knew it but somehow I didn't realize, wasn't intuitive somehow
